server/conf/messages.am

```diff
@@ -135,11 +135,8 @@ content.submittedDate={0} ገብቷል
 # "Get benefits" floating text
 content.benefits=ጥቅማጥቅሞችን ያግኙ
 
-# Long form description of the CiviForm site shown to the applicant. Line 1.
-content.description1=CiviForm መረጃን እንደገና በመጠቀም በአንድ ጊዜ ለበርካታ ጥቅማጥቅሞች እንዲያመለክቱ ያስችለዎታል።
-
-# Long form description of the CiviForm site shown to the applicant. Line 2.
-content.description2=ከታች ማመልከቻ በመምረጥ ይጀምሩ።
+# Long form description of the CiviForm site shown to the applicant.
```

Comment:
**question:** Is the plan to wait to submit this until the translations are added? So this is a placeholder until that happens (here and below)?

Reply:
Oops - yes, that's right. I didn't mean to put you as a reviewer until that happened - just forgot about it. I'll leave you on as a reviewer, but no need to approve until the translations are completed.

Reply:
Sounds good, thanks!